### PR TITLE
Update banner message in V13

### PR DESCRIPTION
**Pull request by diriquelme**
## What does this PR do?
Update “old” spark (https://v13--spark-design-system.netlify.app) Banner Message will read
“For designs launching AFTER July 14, 2021, please reference the latest release of Spark (https://sparkdesignsystem.com).”

### Associated Issue
#3651 

## Please check off completed items as you work.
If a checklist item or section does not apply to your PR then please remove it.

### Documentation
 - [ ] Update Spark Docs

### Browser Testing (current version and 1 prior)
  - [ ] Google Chrome
  - [ ] Google Chrome (Mobile)
  - [ ] Mozilla Firefox
  - [ ] Mozilla Firefox (Mobile)
  - [ ] Microsoft Internet Explorer 11 (only this specific version of IE)
  - [ ] Microsoft Edge
  - [ ] Apple Safari
  - [ ] Apple Safari (Mobile)
  
### Accessibility Testing
  - [ ] Axe browser extension
  - [ ] Jaws Inspect
  - [ ] VoiceOver (iOS) or Talkback (Android)

### Deploy Preview Reviewed and Approved
 - [ ] Design Team
 - [ ] Accessibility Team

### Screenshots
v13_update_banner_message

Update banner on V13 preview
v13_banner_zoom

### CODE REVIEW

File: src/components/ComponentPreview.js [hunk 1]
```diff
@@ -20,10 +20,11 @@ class ComponentPreview extends React.Component {
       maxWidth,
       allowScrolling,
     } = this.props;
-    const iframeURL = `https://html.sparkdesignsystem.com/iframe.html?id=${componentType}-${componentName}`;
+    const iframeURL = `https://v13--spark-sb-html.netlify.app/iframe.html?id=${componentType}-${componentName}`;
```
Comment (afebbraro) [inline src/components/ComponentPreview.js:23]: Why did you change staging to point at the v13 branch for the deploy previews?

File: src/components/layouts/Layout.js [hunk 2]
```diff
@@ -40,38 +40,54 @@ const Layout = ({ children, initialContext, hasSideBar, location }) => {
...
           }
         }
-      }
-    `}
-      render={data => (
+      `}
+      render={(data) => (
         <>
-          <div className="sprk-o-Box docs-c-SkipNav">
+          <div className="sprk-o-Box docs-c-SkipNav sprk-u-BackgroundColor--white">
```
Comment (afebbraro) [inline src/components/layouts/Layout.js:70]: why did you update the background color here?

File: src/components/layouts/Layout.js [hunk 3]
```diff
@@ -40,38 +40,54 @@ const Layout = ({ children, initialContext, hasSideBar, location }) => {
...
+          <div
+            className="sprk-u-BackgroundColor--black sprk-u-Color--white
+            sprk-u-pvm
+            sprk-u-AbsoluteCenter"
+          >
+            <a
+              href="https://www.sparkdesignsystem.com/"
+              className="docs-c-Banner--link sprk-u-mlm"
+            >
+              For designs launching AFTER July 14, 2021, please reference the
```
Comment (afebbraro) [inline src/components/layouts/Layout.js:87]: This whole sentence shouldn't be a link.

File: src/components/layouts/Layout.js [hunk 4]
```diff
@@ -40,38 +40,54 @@ const Layout = ({ children, initialContext, hasSideBar, location }) => {
...
               Skip to Main Content
             </SprkLink>
           </div>
+          <div
+            className="sprk-u-BackgroundColor--black sprk-u-Color--white
+            sprk-u-pvm
+            sprk-u-AbsoluteCenter"
+          >
+            <a
+              href="https://www.sparkdesignsystem.com/"
```
Comment (afebbraro) [inline src/components/layouts/Layout.js:84]: Staging's url is https://www.sparkdesignsystem.com/. So this link wouldn't take them anywhere different.

File: src/pages/using-spark.mdx [hunk 5]
```diff
@@ -43,7 +43,7 @@ import {
           <SprkButton
             variant="secondary"
             element="a"
-            href="https://html.sparkdesignsystem.com/"
+            href="https://v13--spark-sb-html.netlify.app"
```
Comment (afebbraro) [inline src/pages/using-spark.mdx:46]: We don't want to point staging at v13

File: src/scss/_spark-theme.scss [hunk 6]
```diff
@@ -51,20 +51,31 @@ body {
 
 .docs-c-SkipNav {
   position: absolute;
-  top: 0;
+  top: -150px;
```
Comment (afebbraro) [inline src/scss/_spark-theme.scss:54]: Looks like you changed some sass for the skip nav but I didn't see that in the issue. Whats this change for?

**Review by afebbraro — CHANGES_REQUESTED** [review]
I left comments about changes needed.

**Comment by afebbraro** [comment]
Screen Shot 2020-11-19 at 7 18 01 AM
Hey @diriquelme , I took a look again at this and it still needs those comments addressed.

File: src/components/ComponentPreview.js (hunk 1 above)
Comment (diriquelme) [reply src/components/ComponentPreview.js]: Update link to point to sparkdesignsystem.com

Comment (diriquelme) [reply src/components/ComponentPreview.js]: made the PR in the wrong branch, this is not part of the task

File: src/components/layouts/Layout.js (hunk 3 above)
Comment (diriquelme) [reply src/components/layouts/Layout.js]: Updated the message, and added the link to "latest release of Spark" only.

File: src/pages/using-spark.mdx (hunk 5 above)
Comment (diriquelme) [reply src/pages/using-spark.mdx]: made the PR in the wrong branch, this is not part of the task

File: src/scss/_spark-theme.scss (hunk 6 above)
Comment (diriquelme) [reply src/scss/_spark-theme.scss]: made the PR in the wrong branch, this is not part of the task

File: src/components/layouts/Layout.js (hunk 4 above)
Comment (diriquelme) [reply src/components/layouts/Layout.js]: v13 update message should point to the latest Spark release https://sparkdesignsystem.com/ and main update message should point to https://v13--spark-design-system.netlify.app/

File: src/components/layouts/Layout.js (hunk 2 above)
Comment (diriquelme) [reply src/components/layouts/Layout.js:71]: original PR was made in the wrong branch, background color change is not part of this task, v13 already had background color set to white  https://github.com/sparkdesignsystem/spark-design-system/blob/8ca75e2b24930742be3621ce5f8d9e84bdbda962/src/components/layouts/Layout.js#L68

**Comment by diriquelme** [comment]
Hi @afebbraro - original PR was made to the wrong branch and brought edits that were already fixed in v13 and were not part of the "update message in v13 banner" task.

Comments have been resolved and banner in v13 is ready for review. 

Latest commit preview https://deploy-preview-3659--spark-design-system.netlify.app/

**Comment by afebbraro** [comment]
hey @diriquelme, thanks, I took a look this needs some changes design wise and is missing the word "the". Will message you!

**Comment by diriquelme** [comment]
@afebbraro - Banner has been updated to match content approved formatting - https://deploy-preview-3659--spark-design-system.netlify.app/. Thank you for your feedback and comments :)

File: src/components/layouts/Layout.js [hunk 7]
```diff
@@ -40,49 +41,57 @@ const Layout = ({ children, initialContext, hasSideBar, location }) => {
...
-            <a
-              href="https://www.sparkdesignsystem.com/"
-              className="docs-c-Banner--link sprk-u-mlm"
-            >
-              View the latest release of Spark
-            </a>
+            <p className="sprk-u-Color--white">
+              Designs launching
+              <span className="sprk-u-FontWeight--bold sprk-u-mas">after</span>
+              July 14, 2021, please reference
```
Comment (afebbraro) [inline src/components/layouts/Layout.js:87]: Everything looks good besides the year. It says 2021 and should be 2020.
